pkg/api/server.go

```diff
@@ -41,23 +46,43 @@ func NewServer(cfg *config.Config) (*Server, error) {
 		return nil, err
 	}
 	gin.SetMode(gin.ReleaseMode)
-	router := gin.New()
-	router.Use(gin.Recovery(), gin.Logger())
-	apirouter.Mount(router)
+	httpServer := gin.New()
+	httpServer.Use(gin.Recovery(), gin.Logger())
+	apirouter.Mount(httpServer)
 
 	srv := &Server{
-		router:       router,
+		httpServer:   httpServer,
 		httpListener: httpListener,
 	}
-
 	if cfg.EnableProfiling {
 		srv.pprofMu = new(http.ServeMux)
 		srv.pprofMu.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
 		srv.pprofMu.HandleFunc("/debug/pprof/profile", pprof.Profile)
 		srv.pprofMu.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 		srv.pprofMu.HandleFunc("/debug/pprof/trace", pprof.Trace)
 		srv.pprofMu.HandleFunc("/debug/pprof/", pprof.Index)
-		router.GET("/debug/pprof/*profile", gin.WrapF(srv.pprofMu.ServeHTTP))
+		httpServer.GET("/debug/pprof/*profile", gin.WrapF(srv.pprofMu.ServeHTTP))
+	}
+
+	cert, err := tls.LoadX509KeyPair(cfg.CertFilePath, cfg.KeyFilePath)
```

Comment:
Should it be optional? Only enable TLS if the certificate and private key are specified simultaneously.

Reply:
Because `cfg.CertFilePath` and  `cfg.KeyFilePath` have default values.

Reply:
Then you should prepare the default certificate and private key, or it's difficult to start the server in the local environment.